lab-bundle-spec-v1.md

```diff
@@ -0,0 +1,248 @@
+# Qwiklabs Lab Bundle Specification
```

Comment:
Is any part of this file up for review in this PR or is this just here for reference/because files moved around?

Reply:
The latter

Reply:
Well I guess this file's existence and name would be up for discussion here:
- Should we keep the v1 spec in the repo?
- Should it go in a separate folder of "old specs" or stay at the top-level
etc

Reply:
I'm fine leaving it around. After v2 is finalized, we should probably add a message to the top of the file asking people to use v2 and also mention that we're deprecating v1 in q7.

Reply:
Agreed